apps/marketplace/src/components/Sales.tsx

```diff
@@ -40,23 +42,31 @@ const Sales: FunctionComponent<SalesProps> = ({
     <Stack alignItems="center">
       { !!title && (
         <Box mb={ 3.5 }>
-          <Typography
-            fontSize={ ["24px", "24px", "32px"] }
-            textAlign="center"
-            textTransform="uppercase"
-            variant="h3"
-          >
-            { title }
-          </Typography>
+          { typeof title === "string" ? (
+            <Typography
+              fontSize={ ["24px", "24px", "32px"] }
+              textAlign="center"
+              textTransform="uppercase"
+              variant="h3"
+            >
+              { title }
+            </Typography>
+          ) : (
+            title
+          ) }
         </Box>
       ) }
 
       <Grid justifyContent="flex-start" pb={ 1 } rowGap={ 1.5 } container>
-        { !sales.length ? (
+        { !isLoading && !sales.length ? (
           <Box flex={ 1 }>
-            <Typography sx={ { marginTop: 8, textAlign: "center" } }>
-              No songs to display at this time.
-            </Typography>
+            { typeof noResultsContent === "string" ? (
+              <Typography sx={ { marginTop: 8, textAlign: "center" } }>
+                { noResultsContent }
+              </Typography>
+            ) : (
+              noResultsContent
+            ) }
```

Comment:
Is this just prep for some other content? Is design/product working on something to place here instead of the text?